Cython/Compiler/Naming.py

```diff
@@ -13,7 +13,7 @@
 
 temp_prefix       = u"__cyt_"
 
-pyunicode_identifier_prefix = (pyrex_prefix + 'U').encode('ascii')
+pyunicode_identifier_prefix = pyrex_prefix + 'U'
```

Comment:
I also find it much better when the constants in `Naming` are text and not (surprisingly) some of them bytes. `punicodify_name()` is the only place where bytes are handled (temporarily), everything else is text. So this is a good change.